android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/KeymanSettingsActivity.java

```diff
@@ -20,6 +22,7 @@ protected void onCreate(Bundle savedInstanceState) {
     setContentView(R.layout.keyman_settings_layout);
     Toolbar toolbar = (Toolbar) findViewById(R.id.keyman_settings_toolbar);
     toolbar.setTitleTextColor(getResources().getColor(android.R.color.white));
+    toolbar.setTitle(R.string.keyman_settings);
```

Comment:
I take it this was a string we missed earlier?

Reply:
Correct. When the display language was set to Obolo, I noticed the "Settings" menu still showed in Khmer.